.nais/naiserator-dev.yaml

```diff
@@ -102,7 +104,7 @@ spec:
     - name: NAIS_CONTEXT
       value: 'dev'
     - name: NODE_ENV
-      value: "production"
+      value: "development"
```

Comment:
Litt usikker på dette?

Reply:
Tenkte å spør om dette. Virker veldig rart at det skal være `"production"` her 🤔

Reply:
For så vidt enig, men i backend-app'ene så settes det tilsvarende flagget til false bare for lokal kjøring på utviklermaskin (som vi ikke gjør så mye), mens den er true både for dev-gcp og prod-gcp.

Reply:
Hm kan fjerne denne endringen fra denne PR'en. Hvilket flagg tenker du på i backend appene? `KTOR_ENV`?